packages/fhir-router/src/graphql.ts

```diff
@@ -157,16 +157,13 @@ export async function graphqlHandler(req: FhirRequest, repo: FhirRepository): Pr
 
   const dataLoader = new DataLoader<Reference, Resource>((keys) => repo.readReferences(keys));
 
-  let result: any = introspection && introspectionResults.get(query);
```

Comment:
due to added guard above at https://github.com/medplum/medplum/pull/1946/files#diff-edf7ac6d441b8b49c1649bdc9c7da39a333b108c8494d4324eaaf5d19a29eff5L154, `introspection` will always be false here.

Reply:
added the code back, just noticed #1937 where introspection support would be managed through a server config.